game/platforms/SDL/gameSDL.cpp

```diff
@@ -4005,6 +4005,50 @@ char isCommandKeyDown() {
 
 
 
+char isControlKeyDown() {
+    SDLMod modState = SDL_GetModState();
+    
+
+    if( modState & KMOD_CTRL ) {
+        return true;
+        }
+    
+    if( screen->isPlayingBack() ) {
+        // ignore these, saved internally, unless we're playing back
+        // they can fall out of sync with keyboard reality as the user
+        // alt-tabs between windows and release events are lost.
```

Comment:
This comment reads a bit weird since alt is no longer involved in this version of `isCommandKeyDown()`. I would reword this and the below comment as something like "ignore these for consistency with isCommandKeyDown()".